File: hello.rb
# myapp.rb
require 'sinatra'

set :logging, false

get '/' do
  'Hello world!'
end

#
# Install rbenv:
#
# $ brew install rbenv
#
# Install ruby 2.5.0:
#
# $ rbenv install
#
# Get Bundler:
#
# $ gem install bundler
#
# Install gems:
#
# $ bundle install
#
# Run:
#
# $ ruby hello.rb
#

File: hello.go
package main

import (
    "io"
    "net/http"
)

func hello(w http.ResponseWriter, r *http.Request) {
    io.WriteString(w, "Hello world!\n")
}

func main() {
    http.HandleFunc("/", hello)
    http.ListenAndServe(":8000", nil)
}

/*
Install Go:

$ brew install go

Build:

$ go build hello.go

Run:

$ ./hello
*/
